.eslintignore

```diff
@@ -0,0 +1,11 @@
+# Ignore dist files
+client/dist/
+
+# Ignore legacy files
+#client/src/components/GridFieldAddExistingSearchButton/
```

Comment:
I suspect this doesn't belong to this module

Reply:
haha, yup x]